doc/service_reference_card.md

```diff
@@ -45,7 +45,7 @@
   
   * How to block/ban a user
     * To ban users accessing summaries, remove them from `ALLOWED_FOR_GET` in `/var/www/html/apel_rest/settings.py`.
-    * While no explicit means of blocking a POST request currently exists, for a POST request to be authorized it must be listed as an Indigo DataCloud provider. So providers could be banned for the entire Indigo architecture. 
+    * To ban users sending job records, add their HostDN to `BANNED_FROM_POST` in `/var/www/html/apel_rest/settings.py`.
```

Comment:
Possibly add back some of the context about how the banned list relates to the Indigo list and mention why you might want to ban someone.